src/tests/api-tests.ts

```diff
@@ -10813,6 +10813,32 @@ describe('api tests', () => {
     expect(addressEvents.status).toBe(400);
   });
 
+  test('/microblock', async () => {
+    const microblocks = new TestMicroblockStreamBuilder().addMicroblock().build();
+    const microblock = microblocks.microblocks[0];
+    const block = new TestBlockBuilder()
+      .addTx({
+        microblock_hash: microblock.microblock_hash,
+        tx_id: '0xffff',
+      })
+      .addTx({
+        microblock_hash: microblock.microblock_hash,
+        tx_id: '0x1234',
+      })
```

Comment:
These txs should be added to the `TestMicroblockStreamBuilder`, not to the block